website/docs/guides/ide.md

```diff
@@ -53,6 +53,5 @@ Once Metals picks up the project structure that’s created by Scala CLI, basic
 
 Here are a few notes related to IntelliJ support:
 
-- The most significant problem with IntelliJ is that it requires sources to be placed within a directory. Therefore, if you use IntelliJ, we strongly suggest that you place your sources in a directory (like `src`).
```

Comment:
Imho we should warn that Intellij will not precisely recognize what is source and what is not. E.g. in case of:

```
A.scala
B.scala
```

that both contains `Main` object running `scala-cli A.scala` and then open the project will (as far as I understand) add both A.scala and B.scala to Intellij project and highlight that Main is duplicated even though project compiles fine in scala-cli.

Reply:
<img width="1790" alt="image" src="https://user-images.githubusercontent.com/18601388/158363590-b3466a93-2ec8-4578-9cdf-3f8cc173d934.png">
Ah yes, you're right. 
No matter what we pass to `scala-cli`, IntelliJ always operates on whole directories they are in.
I don't think we'll ever be able to fix that because of how IntelliJ itself works.
Alright, lemme explain it in the docs.

Reply:
fixed